packages/amplify-graphql-default-value-transformer/src/graphql-default-value-transformer.ts

```diff
@@ -54,6 +61,17 @@ export class DefaultValueTransformer extends TransformerPluginBase {
     this.directiveMap.get(parent.name.value)!.push(config);
   };
 
+  transformSchema = (ctx: TransformerTransformSchemaStepContextProvider) => {
+    for (const typeName of this.directiveMap.keys()) {
```

Comment:
No need to change this, but you can iterate over both the keys and values at the same time:

```js
for (const [key, value] of map) {
  ...
}
```